<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PopupChatDialog</class>
 <widget class="QWidget" name="PopupChatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">MainWindow</string>
  </property>
  <layout class="QVBoxLayout" name="PopupChatDialogVLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="mainHLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="leftsideframe">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>14</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="leftsideframeVLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="avatarFrameButton">
          <property name="minimumSize">
           <size>
            <width>14</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>14</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>31</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="avatarFrameSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>12</width>
            <height>335</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="avatarframe">
       <property name="maximumSize">
        <size>
         <width>132</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="avatarframeVLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="AvatarWidget" name="avatarWidget">
          <property name="minimumSize">
           <size>
            <width>116</width>
            <height>116</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>116</width>
            <height>116</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="myspacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>61</width>
            <height>141</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="AvatarWidget" name="ownAvatarWidget">
          <property name="minimumSize">
           <size>
            <width>116</width>
            <height>116</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>116</width>
            <height>116</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="chatLayout">
       <property name="topMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="ChatWidget" name="chatWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionClearOfflineMessages">
   <property name="text">
    <string>Clear offline messages</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChatWidget</class>
   <extends>QWidget</extends>
   <header location="global">gui/chat/ChatWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AvatarWidget</class>
   <extends>QLabel</extends>
   <header>gui/common/AvatarWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
